NewForce-Cohort-5/tabloidfullstack-actual-spring
# Jt editcomment

# Description

As a user, i should have the ability to update a comment that I have made on a post. While the user is viewing the list of comments and selects the edit button on the comment they've previously left, they should be presented a form with the comments current contents inside of form inputs. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. Log in as an existing user in the program
2. Click on the post link at the top and then navigate to the details page of one of the post.
3. Click the button to view that post's comments
4. While viewing the comments section, you may have to add a new comment by selecting add comment and saving it. You should be presented the list, now with the comment that you added.
5. To test, select the edit button under your comment and you should see two inputs for your subject and content with the previously input values inside.
6. Make some changes to the value and save the new comment.
7. You should be redirected back to the list of comments with your comment showing the updated values.
8. Please select edit one more time on your comment but this time cancel out of the form and everything should return to the original list.


# Checklist:
